src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="wrapper">
+      <form 
+      class="search__form" 
+      data-qa="big"
+      action="#"
+      method="get"
+      > 
+        <label class="search__label">
+          <img 
+            src="images/Search.svg" 
+            alt="search-icon" 
+            class="img"
+          >
+          <input 
+            type="text"
+            class="search__form-input"
+            placeholder="Try &quot;Los Angeles&quot;"
+            name="search"
+            data-qa="keypress"
+          >
+        </label>
+      </form>
+      <form 
+      class="search__form search__form-sm" 
+      data-qa="small"
+      action="#"
+      method="get"
+      > 
```

Comment:
```suggestion
      <form 
        class="search__form search__form-sm" 
        data-qa="small"
        action="#"
        method="get"
      > 
```

Reply:
Hi! Thanks for your feedback, I have committed the suggested changes, please let me know if it was done correctly (I am just making my first steps on Git;) )